lib/Drush/Psysh/DrushCommand.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * @file
+ * Drush Command class.
+ *
+ * Original author: Justin Hileman
+ *
+ * DrushCommand is a PsySH proxy command which accepts a drush command config
+ * array and tries to build an appropriate PsySH command for it.
+ */
+
+namespace Drush\Psysh;
+
+use Psy\Command\Command as BaseCommand;
+use Symfony\Component\Console\Formatter\OutputFormatter;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Main DrushCommand class.
+ */
+class DrushCommand extends BaseCommand {
+
+  /**
+   * @var array
+   */
+  private $config;
+
+  /**
+   * @var string
+   */
+  private $category;
+
+  /**
+   * DrushCommand constructor.
+   *
+   * This accepts the drush command configuration array and does a pretty
+   * decent job of building a PsySH command proxy for it. Wheee!
+   *
+   * @param array $config
+   *   Drush command configuration array.
+   */
+  public function __construct(array $config) {
+    $this->config = $config;
+    parent::__construct();
+  }
+
+  /**
+   * Get Category of this command.
+   */
+  public function getCategory() {
+    if (isset($this->category)) {
+      return $this->category;
+    }
+
+    $category = $this->config['category'];
+    $title = drush_command_invoke_all('drush_help', "meta:$category:title");
+
+    if (!$title) {
+      // If there is no title, then check to see if the
+      // command file is stored in a folder with the same
+      // name as some other command file (e.g. 'core') that
+      // defines a title.
+      $category = basename($this->config['path']);
+      $title = drush_command_invoke_all('drush_help', "meta:$category:title");
+    }
+
+    return $this->category = empty($title) ? 'Other commands' : $title[0];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this
+      ->setName($this->config['command'])
+      ->setAliases($this->buildAliasesFromConfig())
+      ->setDefinition($this->buildDefinitionFromConfig())
+      ->setDescription($this->config['description'])
+      ->setHelp($this->buildHelpFromConfig());
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $args = $input->getArguments();
+    $first = array_shift($args);
+
+    // If the first argument is an alias, assign the next argument as the
+    // command.
+    if (strpos($first, '@') === 0) {
+      $alias = $first;
+      $command = array_shift($args);
+    }
+    // Otherwise, default the alias to '@self' and use the first argument as the
+    // command.
+    else {
+      $alias = '@self';
+      $command = $first;
+    }
+
+    $return = drush_invoke_process($alias, $command, array_values($args), $input->getOptions(), ['backend' => TRUE]);
```

Comment:
'backend' should be sent in 3rd param. Also, please try to `drush dl page_manager` or something like that twice. You should get prompted about overwrite the 2nd time. If no such prompt happens, try to set `array('interactive' => TRUE)` as $backend_options value (5th param).